pool/config.go

```diff
@@ -53,3 +54,48 @@ type EffectiveGasPrice struct {
 	// MarginFactor is the margin factor percentage to be added to the L2 min gas price
 	MarginFactor float64 `mapstructure:"MarginFactor"`
 }
+
+// BatchConfig represents the configuration of the batch constraints
+type BatchConfig struct {
```

Comment:
sequencer is the component that knows how to deal with batches and the state is the component that controls the batches, why we created this struct here? I recommend moving this config under the State config.

Reply:
We get import cycling if I put it in the `state`. I agree it's better there and I tried first to put it there but now saw the import cycling error and couldn't fix it.